src/main/java/io/appium/java_client/flutter/FlutterDriver.java

```diff
@@ -0,0 +1,62 @@
+package io.appium.java_client.flutter;
+
+import io.appium.java_client.flutter.commands.FlutterCommandParameter;
+import io.appium.java_client.flutter.commands.ScrollParameter;
+import io.appium.java_client.flutter.commands.WaitParameter;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+/**
+ * Interface representing a Flutter driver that extends {@link RemoteWebDriver}.
+ * Provides convenience methods for executing Flutter-specific commands using JavaScriptExecutor.
+ */
+public interface FlutterDriver {
```

Comment:
Please group interfaces by their purpose. For example, in this case we may add following interfaces:
- SupportsWaitingForElemements
- SupportsScrolling
- CanExecuteScripts
- HasWebDriver